front-end/dashboard.js

```diff
@@ -51,6 +52,31 @@ new Vue({
                 // do nothing.
             }
         },
+        cursorHide() {
+            let cursorTimer;
+            document.body.addEventListener(
+                'mousemove',
+                function(e) {
+                    if (e.movementY > 0 || e.movementX > 0) {
+                        window.clearTimeout(cursorTimer);
+                        document.body.style.cursor = null;
+                        if (this.$store.state.settings.cursorHidden) {
+                            cursorTimer = window.setTimeout(function() {
+                                document.body.style.cursor = 'none';
+                            }, this.$store.state.settings.cursorHiddenTimeout);
+                        }
+                    }
+                }.bind(this)
+            );
+            cursorTimer = window.setTimeout(
+                function() {
```

Comment:
if you use `() => {` instead of `function() {` than there is no need to `.bind(this)`. ES6 :smile: 
```suggestion
                () => {
```